curriculum/challenges/english/25-front-end-development/review-javascript-math/6723c463e51a2d9b747d7529.md

```diff
@@ -9,6 +9,246 @@ dashedName: review-javascript-math
+- **Multiplication Operator**: This operator (`*`) is used to calculate the product of one or more numbers. 
+- **Division Operator**: This operator (`/`) is used to calculate the quotient between two numbers
+- **Division By Zero**: If you try to divide by zero, JavaScript will return `Infinity`.
+- **Remainder Operator**: This operator(`%`) returns the remainder of a division. 
+- **Exponentiation Operator**: This operator (`**`) raises one number to the power of another.
+
+## Calculations with Numbers and Strings 
+
+- **Explanation**: When you use the `+` operator with a number and a string, JavaScript will coerce the number into a string and concatenate the two values.  When you use the `-`, `*` or `/` operators with a string and number, JavaScript will coerce the string into a number and the result will be a number. For `null` and `undefined`, JavaScript treats `null` as 0 and undefined as `NaN` in mathematical operations.
+
+```js
+const result = 5 + '10';
+
+console.log(result); // 510
+console.log(typeof result); // string
+
+const subtractionResult = '10' - 5;
+console.log(subtractionResult); // 5
+console.log(typeof subtractionResult); // number
+
+const multiplicationResult = '10' * 2;
+console.log(multiplicationResult); // 20
+console.log(typeof multiplicationResult); // number
+
+const divisionResult = '20' / 2;
+console.log(divisionResult); // 10
+console.log(typeof divisionResult); // number
+
+const result1 = null + 5;
+console.log(result1); // 5
+console.log(typeof result1); // number
+
+const result2 = undefined + 5;
+console.log(result2); // NaN
+console.log(typeof result2); // number
+```
+
+## Operator Precedence 
+
+- **Definition**: Operator precedence determines the order in which operations are evaluated in an expression. Operators with higher precedence are evaluated before those with lower precedence. Values inside the parenthesis will be evaluated first and multiplication/division will have higher precedence then addition/subtraction. If the operators have the same precedence, then JavaScript will use associativity. Associativity is what tells JavaScript whether to evaluate operators from left to right or right to left. For example, the exponent operator is also right to left associative:
+
+```js
+const result = (2 + 3) * 4;
+
+console.log(result); // 20
+
+const result2 = 10 - 2 + 3;
+
+console.log(result2); // 11
+
+const result3 = 2 ** 3 ** 2;
+
+console.log(result3); // 512
+```
+
+## Increment and Decrement Operators
+
+- **Increment Operator**: This operator is used to increase the value by one. The prefix notation `++x` increases the value of the variable first, then returns a new value. The postfix notation `x++` returns the current value of the variable first, then increases it.
+
+```js
+let x = 5;
+
+console.log(++x); // 6
+console.log(x); // 6
+
+
+let y = 5;
+
+console.log(y++); // 5
+console.log(y); // 6
+```
+
+- **Decrement Operator**: This operator is used to decrease the value by one. 
```

Comment:
No examples for this? Or at least another sentence that shows that the decrement operator is `--`